tests/GenerateScriptRegressionTests/GenerateScriptRegressionTests.Tests.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
nit: We usually don't use "regression test" to name test directories and projects. IMO renaming the folder and the project to `GenerateScriptTests` and `GenerateScript.Tests.csproj` would be better.

Reply:
Updated.

Reply:
Could you please refresh and try again? 
https://github.com/dotnet/source-build-reference-packages/pull/689/files